packages/eslint-config-nimble/README.md

```diff
@@ -0,0 +1,65 @@
+# @nimblehq/eslint-config-nimble
+
+This package provides ESLint base rules
+
+## Installation
+
+```
+$ npm install --save-dev @nimblehq/eslint-config-nimble
+```
+
+## Usage
+
+### Standalone
+
+Add `@nimblehq/eslint-config-nimble` to the extends section of your `.eslintrc` configuration file.
+
+```js
+{
+  "extends": [
+      "@nimblehq/eslint-config-nimble"
+  ],
+  "rules": {
+    // Additional, per-project rules...
+  }
+}
+```
+
+### With a framework
+
+Similar to the process above, but usually it requires adding the extra rules for the JS framework:
+
+
+```js
+{
+  "extends": [
+      "@nimblehq/eslint-config-nimble",
+      "plugin:react/recommended",
+      "plugin:vue/recommended"
+  ],
+  "rules": {
+    // Additional, per-project rules...
+  }
+}
+```
+
+This would require defining the required dependencies in the project itself.
+
+## License
+
+This project is Copyright (c) 2014 and onwards.
```

Comment:
```suggestion
This project is Copyright (c) 2014 and onwards Nimble.
```
I think a `Copyright` mention always comes with the author's name, isn't it? 🤔

Reply:
I am also not sure about that 😂 
@nimblehq/engineering-leads Could you give us a suggestion? Other repositories are using this text as well 🙏

Reply:
@malparty is correct. It should also be fixed on https://github.com/nimblehq/git-template

Reply:
Fixed in https://github.com/nimblehq/git-template/pull/24 (I did not create an issue as this is quite minor 🤔 )

Reply:
I also fixed it in https://github.com/nimblehq/eslint-config-nimble/pull/20/commits/4bbfe48a7e89269a56cd3349c6d8b261fda38450